pytrainer/gui/windowmain.py

```diff
@@ -1218,9 +1221,15 @@ def actualize_listview(self,record_list):
             try:
                 _beats = round(i.beats)
             except (ValueError, TypeError) as e:
-                logging.debug("Unable to parse beats for %s", i.beats)
+                logging.debug("Unable to parse beats for %s", i.id)
                 logging.debug(str(e))
                 _beats = 0.0
+            try:
+                _maxbeats = round(i.maxbeats)
+            except (ValueError, TypeError) as e:
+                logging.debug("Unable to parse maxbeats for %s", i.id)
+                logging.debug(str(e))
```

Comment:
Please don't do multiline logging here, just add the exception to the same log line (I know the one you copied this from has the same problem, you can fix that as well while here but you don't need to). Also str(e) isn't needed, logging will handle the string conversion.

Reply:
I added a separate commit to fix the already present multiline logging first.